src/Elastic.Apm/Report/PayloadSenderV2.cs

```diff
@@ -55,6 +55,9 @@ public PayloadSenderV2(IApmLogger logger, IConfigurationReader configurationRead
 			servicePoint.ConnectionLimit = 20;
 
 			_httpClient = new HttpClient(handler ?? new HttpClientHandler()) { BaseAddress = serverUrlBase };
+			_httpClient.DefaultRequestHeaders.UserAgent.Add(new ProductInfoHeaderValue($"elasticapm-{Consts.AgentName}", _service.Agent.Version));
+			_httpClient.DefaultRequestHeaders.UserAgent.Add(new ProductInfoHeaderValue("http-lib",
```

Comment:
Judging by the following node.js example

> elastic-apm-node/2.6.0 elastic-apm-http-client/7.1.1 node/12.2.0

I don't think the intent was to have literally `http-lib` - I think `http-lib` is a placeholder for the actual HTTP client library name that agent uses. In .NET agent case it's `System.Net.Http`, WDTY?